test/unit/BUILD

```diff
@@ -1,21 +1,22 @@
-tests = glob(["*.sh"])
-
 [
     sh_test(
         name = test[0:-3],
         srcs = ["test.sh"],
         args = [
+            "-v",
             test,
         ],
         data = [
             test,
             "rungraphd",
             "//graphd",
+            "//util:gpush",
```

Comment:
I'm considering replacing this with the expanded version (bazel query --output=build ...)

https://gist.github.com/teeler/f2fb4acf433ccae86ce5d878d74e2c39

Maybe not worth it?